File: Introductory Problems/Missing Number/Missing Number.cpp
#include <iostream>
#include <vector>

using namespace std;

void solve(long long int &totalNums, long long int &totalSum)
{
    long long int actualSum = (totalNums * (totalNums + 1)) / 2;
    cout << actualSum - totalSum;
}

int main()
{
    long long int totalNums;
    long long int currentNum;
    long long int totalSum = 0;

    cin >> totalNums;
    vector<long long int> arr;

    for (int i = 0; i < totalNums - 1; i++)
    {
        cin >> currentNum;
        arr.push_back(currentNum);
        totalSum += currentNum;
    }

    solve(totalNums, totalSum);
}

File: Introductory Problems/Missing Number/Missing Number.py
def solve(n, total_sum):
    actual_sum = (n * (n + 1)) // 2
    print(actual_sum - total_sum)


if __name__ == "__main__":
    nums = []
    total_sum = 0
    n = int(input())
    my_list = [int(x) for x in input().split()]
    solve(n, sum(my_list))
